src/createSession.js

```diff
@@ -36,12 +68,18 @@ export default function createSession({
     }
   };
 
+  let onSessionConnected = (event) => {
+    logAnalyticsEvent(apiKey, sessionId, 'react_connected', event.target.connection.connectionId);
```

Comment:
I think we could log variations as well: attempt, success, failure for each action.

Reply:
I didn't add any variation for these logs since it's just for checking if this package is being used or not. I think we can improve it in another PR, but for this scope, I was only looking at adding to see the usage. WDYT?